### PR TITLE
Update spectator document

### PR DESCRIPTION
**Pull request by SilentSpike**
Remove outdated note
Provide the available camera modes in function header preview
Clarify the side filtering mechanism

### CODE REVIEW

File: wiki/feature/spectator.md [hunk 1]
```diff
@@ -106,16 +101,12 @@ In cases where more specific control is required function `ace_spectator_fnc_upd
...
-- Side filter
 - Side list
+- Side filter
 
-The side list is exactly what it sounds like, a list of sides spectatable by the local client. However, unlike spectatable units the side list remains static and can only be updated manually. This is because the side filter is applied on top of the side list whenever the unit list is automatically maintained - meaning the unit list will update if the player changes side or if the side relations change.
-
-Note that the unit whitelist/blacklist also serves to override this side filtering mechanism.
-
-The default side list is `[west,east,resistance,civilian]` and to update it (on the local client) function `ace_spectator_fnc_updateSpectatableSides` can be used:
+The side list is a list of sides *possible* to spectate on the local client, by default this is always `[west,east,resistance,civilian]` and does not change. To update it (on the local client) function `ace_spectator_fnc_updateSpectatableSides` can be used:
```
Comment (bux578) [inline wiki/feature/spectator.md]: `local client. By default`?

Comment (SilentSpike) [inline wiki/feature/spectator.md]: First 2 :+1:

Not sure what you mean by the third

Comment (bux578) [inline wiki/feature/spectator.md]: I've added a `the`

Comment (SilentSpike) [inline wiki/feature/spectator.md]: Added in latest commit, it still reads correctly without it, but I can see why it's not as fluid

Comment (bux578) [inline wiki/feature/spectator.md]: I'm not a native speaker. So my suggestions could be totally wrong ;)

Comment (SilentSpike) [inline wiki/feature/spectator.md]: You're definitely correct in this case :smile: It's in there now.

I think I'm actually going to add some more stuff to the documentation so you'll want to sit on this PR for a little bit.

**Pull request merged**